src/App.tsx

```diff
@@ -58,19 +62,21 @@ function RNGalleryScreenWrapper({navigation}) {
   const Component = RNGalleryList[state.index].component;
   return (
     <View style={styles.container}>
-      <View
+      <TouchableHighlight
         style={{
           backgroundColor: PlatformColor('NavigationViewDefaultPaneBackground'),
           width: 48,
-        }}>
+        }}
+        underlayColor={PlatformColor('NavigationViewDefaultPaneBackground')}
+        onPress={() => navigation.openDrawer()}>
         <TouchableHighlight
           style={styles.menu}
           onPress={() => navigation.openDrawer()}
           activeOpacity={0.5783}
           underlayColor="rgba(0, 0, 0, 0.0241);">
```

Comment:
This is just adding some opacity to the button pressed area. I don't think there's a platform color equivalent unfortunately because the XAML background colors on rest and pressed don't use the same coloring mechanism as touchable opacity, so I had to hard code it.